docs/source/api/apollo-server.mdx

```diff
@@ -599,6 +599,69 @@ The `start` method triggers the following actions:
 1. If your server is a [federated gateway](https://www.apollographql.com/docs/federation/managed-federation/overview/), it attempts to fetch its schema. If the fetch fails, `start` throws an error.
 2. Your server calls all of the [`serverWillStart` handlers](../integrations/plugins-event-reference/#serverwillstart) of your installed plugins. If any of these handlers throw an error, `start` throws an error.
 
+
+## `executeOperation`
```

Comment:
I'm not positive where this function should live within this article, but I don't think it should be between `start` and `stop`...

Maybe just before "Framework-specific middleware function"?

Reply:
Yeah...I did deliberate on where to put this. The main reason I nestled it here is that the `start` method specifically mentions `executeOperation`so I thought that might be a nice tie-in. I agree splitting up `start` and `stop` is awkward though, so I'll move `executeOperation` down after `stop`.